torch/_torch_docs.py

```diff
@@ -7613,10 +7626,13 @@ def merge_dicts(*dicts):
            r"""
 remainder(input, other, *, out=None) -> Tensor
 
-Computes the element-wise remainder of division.
+Computes the element-wise remainder of division with the remainder having the same
+sign as the divisor :attr:`other`.
 
-The dividend and divisor may contain both for integer and floating point
-numbers. The remainder has the same sign as the divisor :attr:`other`.
+.. math::
+    \text{{out}}_i = \text{{input}}_i - rquot * \text{{other}}_i
+
+where :math:`rquot` is the rounded quotient value (rounded towards nearest integer).
```

Comment:
Same comments here about style as in the `fmod` docs

Reply:
Done, thanks!